epub33/rs/index.html

```diff
@@ -2093,15 +2073,38 @@ <h2>Vocabulary association mechanisms</h2>
 						the same URL as the predefined prefix.</p>
 				</dd>
 
-				<dt id="sec-property-datatype">Expanding <code>property</code> Data Types</dt>
```

Comment:
The github review does not allow me to put comments to unchanged parts of the text; in the `<dd>` for prefixes:

whether of it maps to -> whether it maps to